Reference/V9-Config/RuntimeMinificationSettings/index.md

```diff
@@ -21,39 +21,43 @@ This section allows you to configure the runtime minifications (defaults shown),
   }
 }
 ```
+## Use 'in memory' cache
 
-## Use in memory cache
-
-Specifies if Smidge should use an in memory cache or not.
+This setting determines whether Smidge should save it's cached output in memory, or in a file on disk. If set to false, then the folder will be created at the wwwroot of your Umbraco site in a folder called 'Smidge'/
 
 ## Cache buster
 
-Specifies what type of cache buster to use, the options are:
+Specifies mechanism for cache invalidation.
+
+The options are:
 
-* Version - Caches will be busted every time the version string changes.
+* Version - Caches will be busted when your assembly version changes, when the upstream Umbraco version changes and when the version string specified in Configuration changes.
 * AppDomain - Caches will be busted when the app restarts.
-* Timestamp - Caches will be busted based on a timestamp.
+* Timestamp - Caches will be busted based on a timestamp of the bundled files.
 
-## Automatically generated settings
+## Manually changing the Cache buster version
 
-If you use a CacheBuster setting of "Version" you can add an additional configuration option, also called 'Version'  
+If you use a CacheBuster setting of "Version" you can add an additional configuration option, also called 'Version' , which allows you to set a value that you can incremement manually, or via a build server to make sure the version number changes for Smidge and busts the cache.
 
 ```json
 "Umbraco": {
   "CMS": {
     "RuntimeMinification": {
-      "UseInMemoryCache": false,
+      "UseInMemoryCache": true,
       "CacheBuster": "Version",
       "Version": "1234"
     }
   }
 }
 ```
+The actual 'Version' number will not be visible in the url of the assets, this is because it is combined, along with the Umbraco Version from configuration and the your project assembly dll, and then once combined a 'hash' is generated to obscure these details.
+
+in the HTML link thus: ```<link href='/sc/69a3dbf6.1cf661e7.css.d=efb7329470a16d7020272a294742726ebe1da5f1' rel='stylesheet' type='text/css'/>```
```

Comment:
Using `Version` cache buster the output in v9.4.1 is like this

`<link href='/sb/umbraco-backoffice-init-css.css.v7a71f91360259c5f7c3337f152b0df01eeee36f0' rel='stylesheet' type='text/css'/>`

Reply:
done

Reply:
This changes based on the value of [Hosting:Debug](https://our.umbraco.com/documentation/Reference/V9-Config/HostingSettings/#debug).

I'm not sure the emphasis on the hash appearing is useful as long as users are aware of when then cache busting takes place.

However I do wonder if we should change the value to be {{hash}.{configured explicit version value} so the value from config shows in the URL to avoid confusion.

Reply:
> However I do wonder if we should change the value to be {{hash}}.{{configured explicit version value}} so the value from config shows in the URL to avoid confusion.

I think this makes sense!

Reply:
> However I do wonder if we should change the value to be {{hash}}.{{configured explicit version value}} so the value from config shows in the URL to avoid confusion.

Yes, great observation! - it's the 'setting' of an explicit version in the appsettings configuration that then doesn't translate to what's seen on the asset URLs, that causes confusion!